symbulate/results.py

```diff
@@ -188,6 +188,23 @@ def __gt__(self, other):
     def __ge__(self, other):
         return self.apply(lambda x: x >= other)
 
+    def plot(self):
+        raise Exception("Only simulations of random variables (RV) can be plotted, but you simulated from a probability space. you must define a RV on your probability space and simulate it. Then you can call .plot() on those simulations.")    
```

Comment:
Your lines are too long. Please divide up your strings into multiple lines. For example:

```
raise Exception("Only simulations of random variables (RV) can be plotted, "
                "but you simulated from a probability space. you must "
                "define a RV on your probability space and simulate it. ")
etc.
```